src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -504,11 +380,18 @@ public void SetOptions(DacDeployOptions options)
         public void SetGenericDeployOptionProps(PropertyInfo prop, DacDeployOptions options, PropertyInfo deployOptionsProp)
         {
             var val = prop.GetValue(options);
-            var attribute = prop.GetCustomAttributes<DescriptionAttribute>(true).FirstOrDefault();
+            var descriptionAttribute = prop.GetCustomAttributes<DescriptionAttribute>(true).FirstOrDefault();
```

Comment:
will these description and display name attributes always be strings? If so, then it's clearer to have this be string rather than var

Reply:
These attributes are not a string types, and defined as Microsoft.SqlServer.Dac.OptionDescriptionAttribute and Microsoft.SqlServer.Dac.OptionDisplayNameAttribute. :)